clang_tools/install.py

```diff
@@ -162,5 +163,6 @@ def install_clang_tools(version: str, directory: str, overwrite: bool) -> None:
             f"directory is not in your environment variable PATH.{RESET_COLOR}",
         )
     for tool_name in ("clang-format", "clang-tidy"):
-        if install_tool(tool_name, version, install_dir):
+        target_downloaded = install_tool(tool_name, version, install_dir)
+        if target_downloaded or (not target_downloaded and overwrite):
```

Comment:
The condition here `if target_downloaded or (not target_downloaded and overwrite):` is not quite easy to understand not sure if would have an easier way to understand(code).

Reply:
Without a change here, #20 is not fully satisfied. 

`target_downloaded` might deserve a better name. If it is True, then binary was downloaded and installed. if it is False, then binary was not downloaded because it is already installed. Either way, the `install_tool()` function ensures there is a target to symlink.

Another way of doing it would be to ignore the `install_tool()` result.
```py
        install_tool(tool_name, version, install_dir)
        create_sym_link(tool_name, version, install_dir, overwrite)
```
This is still safe because I put an assertion in the `create_sym_link()` to verify a target exe exists.

Reply:
Thanks for asking the question. I've removed the redundant logic.